include/avif/avif.h

```diff
@@ -586,7 +586,7 @@ AVIF_API void avifRGBImageFreePixels(avifRGBImage * rgb);
 // Conversion options.
 typedef enum avifRGBToYUVFlag
 {
-    AVIF_RGB_TO_YUV_DEFAULT = 0, // Uses the first available upsampling filter among:
+    AVIF_RGB_TO_YUV_DEFAULT = 0, // Uses the first available downsampling filter among:
```

Comment:
IMPORTANT: Should we add the sharp yuv filter to the beginning of the list in the next line? I.e., should we prefer sharp yuv by default?

Reply:
I spoke with Frank and he suggested getting it in first and then we can debate whether it should be on by default.
We have to keep in mind that sharpyuv can make images worse if the decoder uses nearest neighbour. However we can also argue that the main consumer of avif files is Chrome and it does support smooth upsampling.
Performance wise I haven't measured the difference, but on paper it should be a bit slower than normal libyuv conversion but I expect the difference to be negligible compared to actual encoding.

Reply:
Thank you for the reply. I agree with your decision.